github/workflows/deploy.yml

```diff
@@ -0,0 +1,24 @@
+---
+name: Deploy Docker
+on:
+  push:
+    branches:
+      - main
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          file: ./docker/Dockerfile
+          push: true
+          tags: ${{ secrets.YP0n }}/food-backend:latest
```

Comment:
i can't see this secret configured.
but if you are going to configure it, please give it a better self descriptive name

Reply:
here is an example:

Reply:
this file has been deleted as unnecessary.